winpty/winpty_wrapper.py

```diff
@@ -11,8 +11,15 @@
 # Local imports
 from .cywinpty import Agent
 
+import os
 import sys
 
+# Refuse to import on Windows 7
+if os.name == 'nt' and sys.getwindowsversion().major == 7:
```

Comment:
Shouldn't this be `sys.getwindowsversion().major <= 7`? Just in case :-)

Reply:
Technical we don't know if Windows 8 works or not either...

Reply:
Let's not rule it out prematurely though.

Reply:
Right. @CAM-Gerlach, could you install `spyder-terminal` on your Win 8 computer to see if it's working fine for you there?